shared/src/data/default-state/patient-templates.ts

```diff
@@ -12,173 +13,1210 @@ const defaultMaleImage: ImageProperties = {
     aspectRatio: 1,
 };
 
-const defaultFemaleImage: ImageProperties = {
-    url: '/assets/female-patient.svg',
-    height: 80,
-    aspectRatio: 1,
-};
+// const defaultFemaleImage: ImageProperties = {
+//     url: '/assets/female-patient.svg',
+//     height: 80,
+//     aspectRatio: 1,
+// };
+
+// returns the change needed to reach the targetHealth from startHealth in the time of phases
+function calculateHealthChange(
+    starthealth: number,
+    targetHealth: number,
+    phases: number
+) {
+    return (targetHealth - starthealth) / (phases * 12 * 60);
```

Comment:
```suggestion
    return (targetHealth - starthealth) / (phases * phaseTime);
```

Or do you want something else here?

Reply:
Yes in that case it's abot tick speed and therefore not in milliseconds

Reply:
Ok, but this relies on the ticks coming every second, which is not guaranteed. @Dassderdie What do you think here?

Reply:
Add TODO comment